lib/airrecord/table.rb

```diff
@@ -197,6 +197,10 @@ def serializable_fields(fields = self.fields)
       }]
     end
 
+    def ==(other)
+      serializable_fields == other.serializable_fields
```

Comment:
It also has the benefit of us know having to rely on `Hash`'s comparison function instead of rolling one our selves